test/functional/posts.js

```diff
@@ -152,7 +152,16 @@ describe("PostsController", function() {
           res.body.should.be.empty
           $should.not.exist(err)
 
-          done()
+          request
+            .post(app.config.host + '/v1/posts/' + post.id + '/like')
+            .send({ authToken: authToken })
+            .end(function(err, res) {
+              err.should.not.be.empty
+              err.status.should.eql(403)
```

Comment:
added these checks. And synced my changes with latest changes with pepyatka:development